before.js

```diff
@@ -13,12 +13,12 @@
  * jQuery(element).on('click', before(5, addContactToList))
  * // => Allows adding up to 4 contacts to the list.
  */
-function before(n, func) {
+const before = (n, func) => {
   let result
   if (typeof func != 'function') {
     throw new TypeError('Expected a function')
   }
-  return function(...args) {
+  return (...args) => {
     if (--n > 0) {
```

Comment:
👆 That shouldn't be an arrow function because we need to forward the `this` in `func.apply(this, args)` below.

Reply:
Done ...